src/main.cpp

```diff
@@ -213,6 +213,7 @@ int main(int argc, char *argv[])
 
 	// Run unit tests
 	if (cmd_args.getFlag("run-unittests")) {
+		porting::attachOrCreateConsole();
```

Comment:
Creating the console here is rather useless because the process exits immediately.

Reply:
indeed. but if you're running MT from an existing console this is also needed for output to appear there at all